examples/swarmion-starter/yarn.lock

```diff
@@ -0,0 +1,11949 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
```

Comment:
Maybe we don't need to include it? I will be generated during the installation process